storm-client/src/jvm/org/apache/storm/daemon/worker/Worker.java

```diff
@@ -215,8 +215,11 @@ private Object loadWorker(IStateStorage stateStorage, IStormClusterState stormCl
                 }
             });
 
+        Integer execHeartBeatFreqSecs = workerState.stormClusterState.isPacemakerStateStore()
+            ? (Integer) conf.get(Config.TASK_HEARTBEAT_FREQUENCY_SECS)
```

Comment:
We might want to update the comment here https://github.com/apache/storm/blob/master/storm-client/src/jvm/org/apache/storm/Config.java#L1678-L1684
since this config is no longer deprecated. and it is used when Pacemaker is in use.

Reply:
Modified Config.